stumpy/core.py

```diff
@@ -1042,7 +1042,7 @@ def compute_mean_std(T, m):
 
 @njit(
     # "f8(i8, f8, f8, f8, f8, f8)",
-    fastmath={"nsz", "arcp", "contract", "afn", "reassoc"}
+    fastmath=config.STUMPY_FASTMATH_FLAGS
```

Comment:
I think we should just go for it and handle #1054 here in this PR if it's okay with you

Reply:
Sure. Will do!